src/Consolonia.Core/Infrastructure/ConsoleWindow.cs

```diff
@@ -137,12 +170,12 @@ public void Show(bool activate, bool isDialog)
 
         public void Hide()
         {
-            throw new NotImplementedException();
+            // toplevel never hides
         }
 
         public void Activate()
         {
-            throw new NotImplementedException();
+            // toplevel is always visible
```

Comment:
In both cases Hide/Activate, should we throw NotSupportedException to indicate it does not success to attempt to hide or to activate?